site/app/controllers/admin/PlagiarismController.php

```diff
@@ -533,62 +616,45 @@ public function saveNewPlagiarismConfiguration(string $new_or_edit, string $grad
             $return_url = $this->core->buildCourseUrl(['plagiarism', 'configuration', 'edit']) . "?gradeable_id={$gradeable_id}&config_id={$config_id}";
         }
 
-        // Check for invalid gradeable/config IDs
         try {
-            $this->verifyGradeableAndConfigAreValid($gradeable_id, $config_id);
+            $config_id = intval($config_id);
```

Comment:
`intval` will never throw an exception. Worst case scenario, it converts the value to `0` for non integer values. I'm guessing you'd want to replace the catch with a `if (!$config_id)`.